bootstrap/internal/controller/agentbootstrapconfig_controller.go

```diff
@@ -553,6 +559,11 @@ func (r *AgentBootstrapConfigReconciler) SetupWithManager(mgr ctrl.Manager) erro
 		).
 		Watches(
 			&hivev1.ClusterDeployment{},
+			// TODO does this depend on the name and namespace for the
+			// ClusterDeployment being the same as the name and namespace of the
+			// ABC? Could we put an owner reference on the CD instead and use
```

Comment:
Yes we definitely need to change a few naming convenction, ClusterImageSet is another example.